.github/workflows/test.yml

```diff
@@ -128,7 +128,7 @@ jobs:
           file: ./testdir/coverage.xml
 
   test-mac:
-    needs: code-quality
+    needs: test-windows
```

Comment:
?

Reply:
the idea is that windows runs first, quickly, *with* `pytest-xdist`.

Reply:
I'm afraid it might not be quicker since you would be running linux and mac after windows which itself takes ~30min so at minimum we are looking at an hour if run sequentially instead of ~40 min.